pkg/proxy/userspace/proxysocket.go

```diff
@@ -213,12 +220,15 @@ func (udp *udpProxySocket) ProxyLoop(service proxy.ServicePortName, myInfo *serv
 					continue
 				}
 			}
+			proxier.recorder.Eventf(proxier.nodeRef, "ReadFromFailure", "UDP: ReadFrom failed, exiting ProxyLoop")
```

Comment:
This should be about the Service or Endpoints, you can omit the "exiting ProxyLoop" part